Samples/TollApp/VSProjects/TollApp.WebApp/App_Data/jobs/continuous/TollApp/TollApp.exe.config

```diff
@@ -33,6 +51,25 @@
       </bindingExtensions>
     </extensions>
   </system.serviceModel>
+  <appSettings>
+    <!-- Service Bus specific app setings for messaging connections -->
+    <!--Event Hub Settings -->
+    <add key="Microsoft.ServiceBus.ConnectionString" value="Endpoint=sb://tollappeventhub.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=ZKhpDMbJ+chHvbD86dO97wfxYIYViB/h9gw0XAocyS0=" />
```

Comment:
Generalize usernames and passwords in this config file so that the user would enter their own.